File: CPP/Program/sampahNonOrganik.cpp
#include <iostream>
#include <string>

using namespace std;

class SampahNonOrganik : public Sampah {
    // Atribut private
    private:
        string jenisNonOrganik; // Jenis sampah (plastik, kaleng, Dll).
    
    public:
        // constructor kosong
        SampahNonOrganik() {
        }

        // Constructor
        SampahNonOrganik(string idSampah, string nama, string berat, 
            string jenisNonOrganik) 
            : Sampah(idSampah, nama, berat) {
            this->jenisNonOrganik = jenisNonOrganik;
        }
        
        // getter setter jenis non organik
        string getJenisNonOrganik() {return jenisNonOrganik;}

        void setJenisNonOrganik(string jenisNonOrganik) {this->jenisNonOrganik = jenisNonOrganik;}
        
    // Destructor
    ~SampahNonOrganik() {
    }
};

File: PYTHON/sampah_non_organik.py
from sampah import Sampah

class SampahNonOrganik(Sampah):
    def __init__(self, id_sampah="", nama="", berat="", jenis_non_organik=""):
        super().__init__(id_sampah, nama, berat)
        self.__jenis_non_organik = jenis_non_organik

    # getter setter jenis non organik
    def get_jenis_non_organik(self):
        return self.__jenis_non_organik

    def set_jenis_non_organik(self, jenis_non_organik):
        self.__jenis_non_organik = jenis_non_organik